generator/main.go

```diff
@@ -94,6 +94,7 @@ func generateConfig(nodes *Node, nameToNode map[string]*Node, logger log.Logger)
 
 var (
 	failOnParseErrors  = kingpin.Flag("fail-on-parse-errors", "Exit with a non-zero status if there are MIB parsing errors").Default("false").Bool()
+	snmpMIBOpts        = kingpin.Flag("snmp.mibopts", "Toggle various defaults controlling MIB parsing, see snmpwalk --help").String()
```

Comment:
could be cool to check if the value is matching what is expected ? Or is it done in a later stage ?

Reply:
The values are dependent on what version of the C library we compile against. It's basically up to the user to figure this out.